src/api/spec/helpers/webui/notification_helper_spec.rb

```diff
@@ -0,0 +1,25 @@
+require 'rails_helper'
+
+RSpec.describe Webui::NotificationHelper, type: :helper do
+  describe '#mark_as_read_or_unread_button' do
+    context 'for unread notification' do
+      let(:notification) { create(:web_notification, delivered: false) }
+
+      it 'contains type as an additional parameter' do
```

Comment:
This is exactly the same description as in the spec for a read notification, although the link isn't the same. This doesn't explain what is different in those 2 specs.

Reply:
Both specs test the same thing. One for `read` notification and one for `unread`. But I've removed the `it` block. It's updated.